test/infer_rt_test.py

```diff
@@ -193,3 +194,41 @@ def test_smoothing_and_causality():
         ),
         "test_smoothing_and_causality",
     )
+
+
+def test_generate_infection_rate_metric_no_region_given():
+    FIPS = []
+    REGIONS = [infer_rt.RegionalInput.from_fips(region) for region in FIPS]
+    df = cli._generate_infection_rate_metric(regions=REGIONS)
+    assert df.empty
+
+
+def test_generate_infection_rate_metric_one_empty():
+    FIPS = [
+        "51017",  # Bath County VA Almost No Cases. Will be filtered out under any thresholds.
+        "51153",  # Prince William VA Lots of Cases
+    ]
+    REGIONS = [infer_rt.RegionalInput.from_fips(region) for region in FIPS]
```

Comment:
nit: generally all caps are indicate global constants [heres a great guide](https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations), can you make these lowercase?

Reply:
done